src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -148,7 +150,9 @@ internal class LincheckClassVisitor(
         mv = WaitNotifyTransformer(fileName, className, methodName, mv.newAdapter())
         mv = ParkingTransformer(fileName, className, methodName, mv.newAdapter())
         mv = ObjectCreationTransformer(fileName, className, methodName, mv.newAdapter())
-        mv = DeterministicHashCodeTransformer(fileName, className, methodName, mv.newAdapter())
+        if (isInTraceDebuggerMode) {
```

Comment:
Don't you always need to apply `DeterministicInvokeDynamicTransformer`?

Reply:
Yes